configuration/items.md

```diff
@@ -261,30 +261,45 @@ FIRST_QUARTER=🌓 First Quarter
 OpenHAB provides a set of [basic icons]({{base}}/addons/iconsets/classic/readme.html) by default.
 However if you wish to use custom icons you need to place them inside the `conf/icons/classic/` folder.
 These icons will be used in all of the openHAB frontends.
-The images must be in `.png` or `.svg` format, and have a name with only small letters and a hyphen or underscore (if required).
-The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
 
-As an example, to use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>`.
+The images must be in `.png` or `.svg` format, consist of only lowercase letters, hyphens `-` or underscores `_` (if required).
+The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files by default.
+
+To use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>` in the Item definition.
 
 ## Dynamic Icons
 You can dynamically change the icon depending on the Item's state.
-You have to provide a default file and one icon file per state with the state's value appended to the icons name.
 
-Example:
+Dynamic icons must meet the following criteria:
+
+* there must be a default icon
+* the name and state part of the icon must consist of all lower case letters, even if the State the icon matches includes uppercase letters
+* the icon selected is based on the mapped value used in the label (i.e. what gets returned by `[MAP(file.map):%s]`, not the Item's raw state.
+
+The name of dynamic icons must meet the following format:
 
-```text
-switch.svg
-switch-off.svg
-switch-on.svg
 ```
+<name>[-<state>].<png or svg>
+```
+Where:
+
+* `<name>` is the name of the icon set
+* `[-<state>]` is the state that particular icon maps to, the icon without the state part is the default
```

Comment:
Where should I file an issue to have icons being mapped to the raw item state? 

If you use labels to localize to your native language or create "nice" names for items, then the default icons do not work any longer. Instead you need to create custom icons. This is defeating the purpose of those.